js/test/threepio.test.js

```diff
@@ -1,23 +1,124 @@
+import * as tf from '@tensorflow/tfjs';
 import Threepio from '../src/threepio';
-import { add } from './fixtures/pytorch';
+import {
+  add,
+  matmul,
+  mean,
+  mul,
+  div,
+  eq,
+  sum,
+  argmax,
+  t,
+  softmax,
+  relu
+} from './fixtures/pytorch';
 const { test, describe, expect } = global; // import jest from global
 
 describe('Threepio', () => {
-  const brokenAdd = {
-    ...add,
-    kwargs: undefined
-  };
   const threepio = new Threepio({});
 
+  const checkAnswer = (result, answer) => {
+    const eq = tf
+      .equal(result, answer)
+      .all()
+      .dataSync();
+    expect(eq).toEqual(new Uint8Array([1]));
+  };
+
   test('returns itself', () => {
     expect(threepio).toBeInstanceOf(Threepio);
   });
 
-  test('translate basic function', () => {
-    threepio.translate('tfjs', add);
+  test('translates add', () => {
+    const translation = threepio.translate('tfjs', add);
+    const result = translation.executeRoutine();
+    const answer = tf.tensor([
```

Comment:
Suggestion: it might be easier to keep `answer` (like `add_answer`) together with the command in fixtures.

Reply:
Suggestion #2: you can even do array of ops/answers in fixtures, and one test to loop over this array :)